eclcli/common/utils.py

```diff
@@ -102,7 +103,7 @@ def get_field(item, field):
 
 
 def get_columns(item):
-    columns = item.keys()
+    columns = list(item.keys())
```

Comment:
In Python 3, `item.keys ()` will be `dict_keys` object which doesn't have `remove` method.  So, it is necessary to convert to `list` object explicitly.